standalone/ansible/roles/postgres_replication/tasks/primary.yml

```diff
@@ -18,6 +18,7 @@
     backup: true
   register: postgres_hba_conf
   with_items: "{{ groups.postgres_secondary }}"
+  become: yes
```

Comment:
Where did we finally land on `become: yes/true`? Is the remaining inconsistency I see intentional?

Reply:
Oops, thanks for spotting! We're sticking to `become: true`. 

Fixed in 96a868f.